ec/internal/converters/extract_endpoint.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types"
 )
 
-// FlattenClusterEndpoint receives a ClusterMetadataInfo, parses the http and
+// ExtractEndpointsToTypes receives a ClusterMetadataInfo, parses the http and
 // https endpoints and returns a map with two keys: `http_endpoint` and
```

Comment:
These functions no longer return a map either, wasn't just the name.

Reply:
I removed the comments. The function signatures are self descriptive.